src/main/java/com/haxwell/apps/questions/utils/DifficultyUtil.java

```diff
@@ -29,42 +30,33 @@ public static int convertToInt(Difficulty d) {
 	}
 	
 	public static int convertToInt(String parameter) {
-		
-		if (parameter.toLowerCase().equals(DifficultyConstants.JUNIOR_STR.toLowerCase()))
-			return 1;
-		if (parameter.toLowerCase().equals(DifficultyConstants.INTERMEDIATE_STR.toLowerCase()))
-			return 2;
-		if (parameter.toLowerCase().equals(DifficultyConstants.SENIOR_STR.toLowerCase()))
-			return 3;
-		if (parameter.toLowerCase().equals(DifficultyConstants.GURU_STR.toLowerCase()))
-			return 4;
-		
-		return 1;
+
+		for (DifficultyEnums de : DifficultyEnums.values()) {
+			if (parameter.toLowerCase().equals(de.getValString().toLowerCase()))
+				return (int)de.getRank();
+		}
+		return 1; //this is an accepted value? Should it be 0 or -1 or a default??
```

Comment:
Yes, the '1' is correct. We want to indicate that the Difficulty is Junior by default. So if `parameter` is not a match, we return Junior